rule/readiness/rule_readiness_test.go

```diff
@@ -0,0 +1,39 @@
+package readiness
+
+import (
+	"testing"
+)
+
+func TestReadinessHealthChecker(t *testing.T) {
+	t.Run("rule readiness probe", func(t *testing.T) {
+		ruleReadinessProbe := NewReadinessHealthChecker()
+
+		if name := ruleReadinessProbe.Name(); name != "rule-first-load" {
+			t.Errorf("Name() did not returned expected name, name = %s", name)
+			return
+		}
```

Comment:
Also these tests should use `require` and `assert` :wink: 
```suggestion
		assert.Equal(t, "rule-first-load", ruleReadinessProbe.Name())
```

Reply:
yep forgot this file sorry